android/java/res/layout/activity_buy_send_swap.xml

```diff
@@ -39,14 +39,10 @@
                     android:layout_width="0dp"
                     android:layout_height="wrap_content"
                     android:layout_weight="2"
-                    android:textSize="12sp"
                     android:textColor="@color/wallet_text_color"
```

Comment:
nit: would be great if you would update this old color with `brave_wallet_day_night_text_color` or BraveWalletTextView style.